images/scripts/fedora-33.bootstrap

```diff
@@ -20,7 +20,7 @@
 set -eux
 
 URL='https://download.fedoraproject.org/pub/fedora/linux/releases/33/Cloud/x86_64/images/'
-IMAGE="$(curl -L -s "$URL" | grep -o '"Fedora-Cloud-Base-[^"]*.qcow2"' | tr -d '"')"
```

Comment:
how about `grep -m 1`?

Bikeshedding at this point, really, but maybe you didn't know about it.

Reply:
Oh, I didn't know about -m, indeed. But it doesn't seem to work with multiple matches in a single line. (The example HTML in the description is in fact all in a single line. I had reformatted it to make it easier to read, but I probably shouldn't have...)

```
$ echo abcabc | grep a -o -m 1
a
a
```